frontend/www/js/omegaup/user/profile.ts

```diff
@@ -45,6 +45,7 @@ OmegaUp.on('ready', () => {
         profile: payload.profile,
         data: payload.extraProfileDetails,
         identities: payload.identities,
+        apiTokens: payload.apiTokens,
```

Comment:
De acuerdo a los comentarios anteriores, `apiTokens`  estará almacenado en el `commonPayload`, así que habrá que cambiar esta línea 
```suggestion
        apiTokens: commonPayload.apiTokens,
```

Además de agregar :
```ts
  const commonPayload = types.payloadParsers.CommonPayload();
```
justo después de donde se declara el `payload`